src/content/docs/core-concepts/optimizing-arc-integration.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: ARCs in Enabling Platforms
+title: Optimizing ARC Integration
```

Comment:
This was supposed to outline a somewhat specific use-case / type of integration in "core facilities" or other service-type labs. 
It's not necessarily, the typical first step for "ARC integration". 

That said, it would probably help to have a similar guide from a lab / group perspective.

Reply:
Can you give more insight on this topic

Reply:
This 

`title: Optimizing ARC Integration` 

adresses different people / a different use-case than this 

`title: ARCs in Enabling Platforms`

Reply:
I might be not knowledgeable enough in this topic, but i do not know what "enabling platform" is meant to be. So i tried to aim for a more expressive title. Something you can read as title in the sidebar and somewhat know what the page will contain. Do you possibly have some suggestions?

Reply:
No, currently no. And I don't want to hold back on the other changes.